src/explore-education-statistics-admin/src/services/methodology/types.ts

```diff
@@ -9,3 +9,9 @@ export interface MethodologyStatusPublication {
   id: string;
   title: string;
 }
+
+export interface CreateMethodologyRequest {
+  title: string;
+  publishScheduled: Date;
```

Comment:
We still using this, as most usages in the backend are commented out

Reply:
As above comments :)